server/sonar-server/src/test/java/org/sonar/server/user/ws/ChangePasswordActionTest.java

```diff
@@ -54,13 +53,14 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
+import static org.sonar.api.utils.System2.SYSTEM;
 
 public class ChangePasswordActionTest {
 
   static final Settings settings = new Settings().setProperty("sonar.defaultGroup", "sonar-users");
 
   @Rule
-  public DbTester dbTester = DbTester.create(System2.INSTANCE);
+  public DbTester dbTester = DbTester.create(SYSTEM);
```

Comment:
considering static import (only), I would have gone further and named the constant `SYSTEM2`